### DISCUSSION TITLE
number of visible options

### DISCUSSION THREAD

**Discussion by rere16** [JedWatson/react-select#5525] [Q&A]
Hi,

Is there an attribute to specify the number of visible options in a drop-down list ? Like the attribute 'size' for a HTML element 'select'.

```
 <label for="cars">Choose a car:</label>

<select name="cars" id="cars" size="3">
  <option value="volvo">Volvo</option>
  <option value="saab">Saab</option>
  <option value="opel">Opel</option>
  <option value="audi">Audi</option>
</select> 
```
Thx

**Comment by manjushsh** [comment]
Hi @rere16, If you want to limit number of options shown, please [check this discussion with solution by @ebonow ](https://github.com/JedWatson/react-select/issues/4516#issuecomment-868106006)
